### PR TITLE
datastore documentation and manual dsr fulfilment

### PR DESCRIPTION
**Pull request by conceptualshark**
Closes DOX-164, DOX-163

### Code Changes

* Added additional docs information + new images for creating a datastore, creating a manual connection, and fulfilling a manual request. 

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation Updated:
  * [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  * [ ] documentation issue created (tag docs-team to complete issue separately)
* [x] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`

### CODE REVIEW

File: docs/fides/docs/ui/datastores.md [hunk 1]
```diff
@@ -22,9 +22,46 @@ All currently configured datastores will appear in the paginated Datastore Conne
...
-Each configured datastore includes an option to `Test` its connection. Fidesops will record the last tested time to the datastore's card, and update the current connection status.
+To add a new datastore connection, select "**Create New Connector**" from the Datastore Connections panel. You will be directed to a list of all available connection options, including adding [manual connectors](#manual-connections).
+
+![new connection](../img/admin_ui/new_connection.png)
+
+Search for and select your desired connector. The Datastores UI will assist in adding and configuring your new connection. 
+
+### Automated connections
+
+Fides automatically includes your third-party connections when processing [privacy requests](../getting-started/privacy_requests.md). Once you have selected a connection type, the UI allows you to describe your connection's configuration information, which includes any necessary fields for accessing and updating third-party data.
```
Comment (seanpreston) [inline docs/fides/docs/ui/datastores.md:35]: It'd be good to clarify "third-party connections" here, I know elsewhere they're referred to as SaaS connectors for instance. I prefer this naming.

File: docs/fides/docs/ui/datastores.md [hunk 2]
```diff
@@ -22,9 +22,46 @@ All currently configured datastores will appear in the paginated Datastore Conne
...
+
+### Manual connections
+
+Manual connections are available for any data that cannot be processed automatically. Examples of manual connections might include physical storage locations, or third-party services without accessible APIs. Fides will pause processing a privacy request at any manual connection, and wait for administrator input before continuing.
+
+To add a manual connection, select **Manual connection** in the "Show all connectors" dropdown. 
+
+![manual connection](../img/admin_ui/manual_connection.png)
+
+One selected, Fides allows you to name and describe your manual connection, as well as provide a list of connection owners. These owners will be contacted by email when needed to respond to a privacy request with a manual component.
```
Comment (seanpreston) [inline docs/fides/docs/ui/datastores.md:49]: > One selected ...

should read

> Once selected ...

File: docs/fides/docs/ui/datastores.md [hunk 3]
```diff
@@ -22,9 +22,46 @@ All currently configured datastores will appear in the paginated Datastore Conne
...
-Each configured datastore includes an option to `Test` its connection. Fidesops will record the last tested time to the datastore's card, and update the current connection status.
+To add a new datastore connection, select "**Create New Connector**" from the Datastore Connections panel. You will be directed to a list of all available connection options, including adding [manual connectors](#manual-connections).
+
+![new connection](../img/admin_ui/new_connection.png)
+
+Search for and select your desired connector. The Datastores UI will assist in adding and configuring your new connection. 
+
+### Automated connections
+
+Fides automatically includes your SaaS connectors when processing [privacy requests](../getting-started/privacy_requests.md). Once you have selected a connection type, the UI allows you to describe your connection's configuration information, which includes any necessary fields for accessing and updating third-party data.
```
Comment (pattisdr) [inline docs/fides/docs/ui/datastores.md:35]: This feels like it is too specific to SaaS Connectors, as if it is the only type of connector available on this screen, but there are database connectors too

Comment (conceptualshark) [reply docs/fides/docs/ui/datastores.md]: I think we're covering SaaS Connectors in this tab, and db connectors in Datastores? Just going off of sean's comment above to update this to call them SaaS Connectors, like elsewhere.

**Pull request closed without merging**